x11-wm/i3/Manifest

```diff
@@ -1,3 +1,4 @@
 DIST i3-4.20.1.tar.xz 1284984 BLAKE2B a7943b3bc680eb3df596b008d751d034fc6ccf2b6891db731b9af5164fdcfc2872c0756c81c3f11f94a30aa9bdc4ecf4b47d20022a214bb6eb6c06fd78458fb3 SHA512 c0f6b991d46180ca470cbb7ab4cf5492552beb9dbf2631c61c8d92d0647106885481962381682e9129f3993d70bc6e5de506da1d32c3710fca64a66d51e8c8ce
 DIST i3-4.21.1.tar.xz 1297400 BLAKE2B 44661bd9edf0bee384b8be88cbf2361bb2ad26fdbaee9a2e010ebdee662b31129057a1a183f46bef0839a545f63c70d00903b0ec4693c1bc9390c84d7a78ac44 SHA512 9656b302d6654f38c48bfa1d97a89eaf3ed395027039b6903005d81a9e95d2c4aeba561cf8d1aa36962ba9975996f133ce4298132cddde0f70ba36f57af3197d
 DIST i3-4.21.tar.xz 1294968 BLAKE2B cf53e0087cb4dabc8eefa8430cc5f7057de927ef49f8172a05c62d690ca60e5d20733da4f3a6230d311bf54a94f117d21665e3d2b39b7f45814e3b0462e12efe SHA512 db56d93850b0cbc8352f5762d9596d38a641e4cf7330cf0afd3778c41a8ed140bbbed3e5419023b1bf3dcc1c470da40018aac9bf684b18d02c12ffe886d29193
```

Comment:
Drop it (not 4.21.1 as there is a stablereq here: https://bugs.gentoo.org/890218)